compiler/noirc_evaluator/src/ssa.rs

```diff
@@ -164,6 +164,7 @@ pub struct ArtifactsAndWarnings(pub Artifacts, pub Vec<SsaReport>);
 /// something we take can advantage of in the [secondary_passes].
 pub fn primary_passes(options: &SsaEvaluatorOptions) -> Vec<SsaPass> {
     vec![
+        SsaPass::new(Ssa::expand_signed_checks, "expand signed checks"),
```

Comment:
This name is temporary but not sure on what would be best. The only thing that comes to mind is `expand_signed_overflow_checks`, but I'm open to suggestions.